package/src/plugins/add-virtual-imports.ts

```diff
@@ -5,10 +5,9 @@ export const addVirtualImportsPlugin = definePlugin({
 	name: "addVirtualImports",
 	hook: "astro:config:setup",
 	implementation: ({ config, updateConfig }, { name }) => {
-		let counter = 1;
 		return (imports: Parameters<typeof addVirtualImports>[0]["imports"]) => {
 			addVirtualImports({
-				name: `${name}-${counter++}`,
+				name,
```

Comment:
Why is that?

Reply:
The functionality for incrementing plugin names now exists inside `addVitePlugin` so this is extraneous